src/lib/mongoBackend/dbConstants.h

```diff
@@ -76,11 +76,11 @@
 #define CSUB_CONDITIONS         "conditions"
 #define CSUB_CONDITIONS_TYPE    "type"
 #define CSUB_CONDITIONS_VALUE   "value"
-#define CSUB_CONDITIONS_EXPR    "expression"
-#define CSUB_CONDITIONS_Q       "q"
-#define CSUB_CONDITIONS_GEOM    "geometry"
-#define CSUB_CONDITIONS_COORDS  "coords"
-#define CSUB_CONDITIONS_GEOREL  "georel"
+#define CSUB_EXPR               "expression"
```

Comment:
I have take the opportunity of this PR to do a global renaming for names closer to how the fields are in the DB model.

NTC